prow/config/plugins.yaml

```diff
@@ -40,8 +40,10 @@ goose:
 
 label:
   additional_labels:
-    - kind/pr
-    - kind/issue
+    # Custom kind/* labels for evolution repository
+    - kind/sandbox
```

Comment:
@kris-nova @leogr this PR is blocked from two things:

- [ ] `contrib` rename to `evolution`
- [ ] decide what label do we want to identify/automate the level (and create ISSUE_TEMPLATEs accordingly)

I can proceed with the rename, but then I would not be able to merge this PR (which instructs prow about the name change) because blocked by point 2.

Reply:
As discussed [here](https://github.com/falcosecurity/contrib/issues/19#issuecomment-628548388), I thought we already decide about the following labels:
- `kind/sandbox`
- `kind/incubating`
- `kind/officialsupport`
(they just match the status name, as documented [here](https://github.com/falcosecurity/contrib/pull/24/files#diff-04c6e90faac2675aa89e2176d2eec7d8))

Regarding the ISSUE_TEMPLATEs, I can fix them.

Reply:
- [x] [labels added](https://github.com/falcosecurity/contrib/labels)
- [x] Issue templates added to [contrib#24](https://github.com/falcosecurity/contrib/pull/24)